3rdparty/TIPS/download_tips.py

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# Download TIPS 2021 data from Zenodo.
+#
+# Sometimes the download fails for no apparent reason. If that happens, wait
+# a few seconds and rerun the download script. It will continue from the point
+# where the error occurred.
+#
+# Needs the zenodo_get module: pip install --user zenodo_get
+
+python -m zenodo_get -R 5 -m -o TIPS2021 10.5281/zenodo.4708098
```

Comment:
@olemke This seems to create a TIPS2021 folder locally?  Is it possible to add also a local `.gitignore` to ignore this folder?

Reply:
Oops, it started as python script then ended up as a bash script. Forgot to change the extensions.
I added the download folder to `.gitignore`